node/src/types/chainspec/core_config.rs

```diff
@@ -84,6 +84,17 @@ pub struct CoreConfig {
 
     /// Which consensus protocol to use.
     pub consensus_protocol: ConsensusProtocolName,
+
+    /// The split in finality signature rewards between block producer and participating signers.
+    #[data_size(skip)]
+    pub finders_fee: Ratio<u64>,
+
+    /// The proportion of baseline rewards going to reward finality signatures specifically.
+    #[data_size(skip)]
+    pub finality_signature_proportion: Ratio<u64>,
+
+    /// Lookback interval indicating which past block we are looking at to reward.
+    pub rewards_lag: u32,
```

Comment:
Since block height is a `u64`, this might work better as a `u64`, too. I mean, I don't think we'll ever have a lag greater than 4 billion blocks, but it will avoid doing typecasting later down the line :sweat_smile: